modules/ding_paragraphs/js/ding_paragraphs_form.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file
+ *
+ * Disables dragging in paragraphs when any paragraphs are open.
```

Comment:
During tests users experienced loss of data when dragging open paragraphs around, i believe it was a conflict in the js between wysiwyg contrib and paragraphs contrib modules. Since dragging the open paragraphs around was in no way a pleasant user experience anyway this solution was accepted.

Reply:
Added comment to code: https://github.com/ding2/ding2/pull/1406/commits/efacdd5eb99cab7414082b560956e4c0b4e7cea4